README.md

```diff
@@ -36,25 +34,26 @@ example of Ioc in NInject for statsd publisher with values from config:
         .WithConstructorArgument("prefix", statsdPrefix);
 
 ```
-####
-Timing with the interface. Given an existing instance of `IStatsDPublisher` you can do:
+
+#### Example of using the interface
+
+Given an existing instance of `IStatsDPublisher` called `stats` you can do for e.g.:
 
 ```csharp
+		stats.Increment("DoSomething-Attempt");
 		var stopWatch = Stopwatch.StartNew();
-
         var success = DoSomething();
 
 		stopWatch.Stop();
 		if (success)
         {
-			stats.Timing("DoSomething.Success", stopWatch.Elapsed);
+			stats.Timing("DoSomething-Success", stopWatch.Elapsed);
```

Comment:
I'd keep the `.` here. Segments with more than one piece of logical data are less flexible.